### ISSUE TITLE
Install error on Ubuntu 16.04

### ISSUE THREAD

**Issue by xyangk** [aymara/lima#61]
I download 7 deb files (ubuntu 14.04 packages), I tryed `sudo dpkg -i lima*.deb`, but failed.(I have installed dependencies.)

```
xiao@xiao-ThinkPad-E460:~/Downloads/lima$ sudo dpkg -i lima*.deb 
(Reading database ... 275673 files and directories currently installed.)
Preparing to unpack limacommon-2.1.1495084954-d2ea4866-devel.deb ...
Unpacking limacommon-devel (2.1.1495084954-d2ea4866) over (2.1.1495084954-d2ea4866) ...
Preparing to unpack limacommon-2.1.1495084954-d2ea4866-runtime.deb ...
Unpacking limacommon-runtime (2.1.1495084954-d2ea4866) over (2.1.1495084954-d2ea4866) ...
Selecting previously unselected package limalinguisticdata-common.
Preparing to unpack limalinguisticdata-2.1.1495084954-d2ea4866-common.deb ...
Unpacking limalinguisticdata-common (2.1.1495084954-d2ea4866) ...
Selecting previously unselected package limalinguisticdata-eng.
Preparing to unpack limalinguisticdata-2.1.1495084954-d2ea4866-eng.deb ...
Unpacking limalinguisticdata-eng (2.1.1495084954-d2ea4866) ...
Selecting previously unselected package limalinguisticdata-fre.
Preparing to unpack limalinguisticdata-2.1.1495084954-d2ea4866-fre.deb ...
Unpacking limalinguisticdata-fre (2.1.1495084954-d2ea4866) ...
Selecting previously unselected package limalinguisticprocessing-devel.
Preparing to unpack limalinguisticprocessing-2.1.1495084954-d2ea4866-devel.deb ...
Unpacking limalinguisticprocessing-devel (2.1.1495084954-d2ea4866) ...
Selecting previously unselected package limalinguisticprocessing-runtime.
Preparing to unpack limalinguisticprocessing-2.1.1495084954-d2ea4866-runtime.deb ...
Unpacking limalinguisticprocessing-runtime (2.1.1495084954-d2ea4866) ...
Setting up limacommon-devel (2.1.1495084954-d2ea4866) ...
Setting up limacommon-runtime (2.1.1495084954-d2ea4866) ...
dpkg: dependency problems prevent configuration of limalinguisticdata-common:
 limalinguisticdata-common depends on limacommon; however:
  Package limacommon is not installed.
 limalinguisticdata-common depends on limalinguisticprocessing; however:
  Package limalinguisticprocessing is not installed.

dpkg: error processing package limalinguisticdata-common (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of limalinguisticdata-eng:
 limalinguisticdata-eng depends on limacommon; however:
  Package limacommon is not installed.
 limalinguisticdata-eng depends on limalinguisticprocessing; however:
  Package limalinguisticprocessing is not installed.

dpkg: error processing package limalinguisticdata-eng (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of limalinguisticdata-fre:
 limalinguisticdata-fre depends on limacommon; however:
  Package limacommon is not installed.
 limalinguisticdata-fre depends on limalinguisticprocessing; however:
  Package limalinguisticprocessing is not installed.

dpkg: error processing package limalinguisticdata-fre (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of limalinguisticprocessing-devel:
 limalinguisticprocessing-devel depends on limacommon; however:
  Package limacommon is not installed.

dpkg: error processing package limalinguisticprocessing-devel (--install):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of limalinguisticprocessing-runtime:
 limalinguisticprocessing-runtime depends on limacommon; however:
  Package limacommon is not installed.

dpkg: error processing package limalinguisticprocessing-runtime (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 limalinguisticdata-common
 limalinguisticdata-eng
 limalinguisticdata-fre
 limalinguisticprocessing-devel
 limalinguisticprocessing-runtime
```

**Comment by pquentin** [comment]
I'm not working on lima, but I think the issue here is that you're trying to install all the debs at once, while they depend on each other. This means you need to install limacommon first, then limalinguisticprocessing, etc. with separate `sudo dpkg -i` calls.

Actually, it looks like limacommon got installed correctly here, but the others were not. Which means you should be able to install limalinguisticprocessing now.

Hope that helps.

**Comment by kleag** [comment]
In fact, the package generation was changed recently to split packages into runtime and devel, but the dependencies have not been between packages have not been updated. Thus limalinguisticprocessing still searches for limacommon while it depends now on limalinguisticprocessing-runtime.

I am currently testing the correction before pushing.

**Comment by kleag** [comment]
@xyangk, please note that by using Ubuntu 14 packages on Ubuntu 16, it could work or not depending on the fact that other dependencie (Qt, boost…) are binary compatible or not.

**Comment by xyangk** [comment]
@pquentin Thank you for your help, @kleag thank you, I'll try again after you done.

**Comment by kleag** [comment]
Well, the dependency problem is solved but I just saw that the runtime packages are empty. They contain no file… I have to correct that before you can test again.

**Comment by xyangk** [comment]
(ง •̀_•́)ง

**Comment by kleag** [comment]
@xyangk , the packaging is now corrected.

But I just checked on my system (KDE neon, based on Ubuntu 16.04), and it did not work as Lima for Ubuntu 14.04 depends on boost 1.54 while there is boost 1.58 on Ubuntu 16.04.

I suggest to either install Lima either in a virtual machine running Ubuntu 14.04 or compiling from source.

We should probably find a CI service running Ubuntu 16.04 or creating a docker or again creating a self contained package like flatpack or another one.
